src/ResourceManager/Sql/Commands.Sql/TransparentDataEncryption/Model/AzureRmSqlManagedInstanceTransparentDataEncryptionCertificateModel.cs

```diff
@@ -0,0 +1,47 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+namespace Microsoft.Azure.Commands.Sql.TransparentDataEncryption.Model
+{
+    /// <summary>
+    /// Represents the core properties of an Azure Sql Database Transparent Data Encryption Certificate
+    /// </summary>
+    public class AzureRmSqlManagedInstanceTransparentDataEncryptionCertificateModel
+    {
```

Comment:
Why do you need the entire class? Related cmdlet returns bool and piping scenarios are not provided for the ManagedInstance.

Reply:
We plan to implement the piping scenarios for ManagedInstance in future when possible. Hence keeping the implementations separate.